README.md

```diff
@@ -115,6 +115,8 @@ Inspired by the [awesome](#more-awesome) list thing. Feel free to <a href="https
 * [Elm Conf 2016](https://www.youtube.com/channel/UCOpGiN9AkczVjlpGDaBwQrQ) - All talks from elm-conf 2016
 * [Amitai Burstein: Frontend with Guarantees](https://www.youtube.com/watch?v=FgaoOgJ5CAU) - A talk from You Gotta Love Frontend 2016
 * [Jessica Kerr: Adventures in Elm](https://www.youtube.com/watch?v=cgXhMc8M4X4) - A talk about the combination of functional programming with Elm at GOTO Chicago 2016.
+* [Aaron VonderHaar: Elm live coding videos ](https://www.youtube.com/user/gruen0) - Lots of live coding videos on youtube.
```

Comment:
That is better, but ElmLive playlist is missing a lot Codevember videos. And not all, are in the Codevember playlist (https://www.youtube.com/playlist?list=PLDA4wlOlLJvXAEsJDje4hdLazsihZiQNf). Do you have any idea how to get them all in the nicest way possible?